source/extensions/filters/network/redis_proxy/command_splitter_impl.h

```diff
@@ -45,18 +45,18 @@ class Utility {
  * All command level stats. @see stats_macros.h
  */
 // clang-format off
-#define ALL_COMMAND_STATS(COUNTER, HISTOGRAM)                                                      \
+#define ALL_COMMAND_STATS(COUNTER)                                                                 \
   COUNTER(total)                                                                                   \
   COUNTER(success)                                                                                 \
-  COUNTER(error)                                                                                   \
-  HISTOGRAM(latency)
```

Comment:
what happened to this?

Reply:
This extension relied on runtime decision for microsecond vs millisecond precision of this histogram and since it's no longer possible to change the unit at run time I moved the declaration out of the macro to the decision point, see `InstanceImpl::addHandler` in `source/extensions/filters/network/redis_proxy/command_splitter_impl.cc`.